code/modules/surgery/organs/ears.dm

```diff
@@ -133,3 +133,27 @@
 	desc = "The robust ears of a bronze golem. "
 	damage_multiplier = 0.1 //STRONK
 	bang_protect = 1 //Fear me weaklings.
+
+/obj/item/organ/ears/cybernetic
+	name = "cybernetic ears"
+	icon_state = "ears-c"
+	desc = "a basic cybernetic designed to mimic the operation of ears."
+	damage_multiplier = 0.9
+	organ_flags = ORGAN_SYNTHETIC
+
+/obj/item/organ/ears/cybernetic/upgraded
+	name = "upgraded cybernetic ears"
+	icon_state = "ears-c-u"
+	desc = "an advanced cybernetic ear, surpassing the performance of organic ears"
+	damage_multiplier = 0.5
+	bang_protect = 1
+
+/obj/item/organ/ears/cybernetic/emp_act(severity)
+	. = ..()
+	if(. & EMP_PROTECT_SELF)
+		return
+	switch(severity)
+		if(1)
```

Comment:
Defines please.

Reply:
This is ripped straight from liver cybernetics code. Technically doesn't even have to exist, I'm not sure why they made a switch case for two cases.